packages/components/Avatar/src/Avatar.styling.ts

```diff
@@ -157,6 +157,20 @@ export const stylingSettings: UseStylingOptions<AvatarProps, AvatarSlotProps, Av
       },
       ['size', 'ringColor', 'ringBackgroundColor', 'ringThickness', ...borderStyles.keys],
     ),
+    imageRing: buildProps(
```

Comment:
it's interesting for me if it's possible to use the same ring slot and just to change tokens

Reply:
Oh, I see there's an image slot, so it's good to use a separate slot